.github/workflows/docs-language-linter.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: Docs Language Linter
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+      - '**.md'
+
```

Comment:
It should run also when the configuration changes, i.e.:
* `.vale.ini`
* `.github/styles/**`
* `.github/workflows/docs-language-linter.yaml`

Reply:
Missing the first two

Reply:
@jond01 added